File: src/main/java/no/nav/tag/sykefravarsstatistikk/api/domene/SlettOgOpprettResultat.java
package no.nav.tag.sykefravarsstatistikk.api.domene;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class SlettOgOpprettResultat {

    private int antallRadSlettet;
    private int antallRadOpprettet;

    public void add(SlettOgOpprettResultat resultat) {
        antallRadOpprettet = antallRadOpprettet + resultat.getAntallRadOpprettet();
        antallRadSlettet = antallRadSlettet + resultat.getAntallRadSlettet();
    }
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/importering/domene/SlettOgOpprettResultat.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.importering.domene

data class SlettOgOpprettResultat(var antallRadSlettet: Int = 0, var antallRadOpprettet: Int = 0) {

    companion object {

        fun tomtResultat(): SlettOgOpprettResultat {
            return SlettOgOpprettResultat(0, 0)
        }
    }
}
